jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/LocalRegistry.java

```diff
@@ -68,19 +69,11 @@ public void start() throws IOException, InterruptedException {
             Arrays.asList(
                 "docker", "run", "--rm", "-d", "-p", port + ":5000", "--name", containerName));
     if (username != null && password != null) {
-      // Generate the htpasswd file to store credentials
-      String credentialString =
-          new Command(
-                  "docker",
-                  "run",
-                  "--rm",
-                  "--entrypoint",
-                  "htpasswd",
-                  "registry:2.7.0", // TODO: correctly fix this when using latest
-                  "-Bbn",
-                  username,
-                  password)
-              .run();
+      // since registry:2 no longer contains htpasswd, generate the htpasswd file manually
```

Comment:
I think we can just say
```suggestion
      // Equivalent of "$ htpasswd -nbB username password".
```
There's no reason to use or consider `registry:2` at all. The image just happened to have `htpasswd` installed.